fastlane_core/lib/fastlane_core/ui/interface.rb

```diff
@@ -118,9 +118,11 @@ class FastlaneCrash < StandardError
     # raised from user_error!
     class FastlaneError < StandardError
       attr_reader :show_github_issues
+      attr_reader :error_info
```

Comment:
I agree with you, *error_*  is redundant inside the exception, good catch.
The reason behind error_Info was to follow with [SpaceShip similar error class](https://github.com/fastlane/fastlane/blob/e02b5d15ec6606bad0ad954b0fb62fea144f39b2/spaceship/lib/spaceship/client.rb#L72) which also expose property named `error_info`